examples/python/gateway_example.py

```
@@ -8,13 +8,34 @@
 from praisonaiagents import Agent, GatewayConfig, SessionConfig
 
 # Configure the gateway
+from pydantic import BaseModel, Field
+
+class ServerConfig(BaseModel):
+    host: str = "0.0.0.0"
```

Ruff (0.15.21)

[error] Possible binding to all interfaces

(S104)